wpilibj/src/athena/cpp/lib/CANJNI.cpp

```diff
@@ -93,13 +95,14 @@ Java_edu_wpi_first_wpilibj_can_CANJNI_FRCNetCommCANSessionMuxReceiveMessage(
   FRC_NetworkCommunication_CANSessionMux_receiveMessage(
       messageIDPtr, messageIDMask, buffer, &dataSize, timeStampPtr, &status);
 
-  CANJNI_LOG(logDEBUG) << "Message ID " << std::hex << *messageIDPtr;
+  CANJNI_LOG(logDEBUG) << "Message ID ";
+  CANJNI_LOG(logDEBUG).write_hex(*messageIDPtr);
 
   if (logDEBUG <= canJNILogLevel) {
-    std::ostringstream str;
-    str << std::setfill('0') << std::hex;
+    llvm::SmallString<128> buf;
+    llvm::raw_svector_ostream str(buf);
     for (int32_t i = 0; i < dataSize; i++) {
-      str << std::setw(2) << (int)buffer[i] << ' ';
+      str.write_hex(buffer[i]) << ' ';
```

Comment:
This doesn't quite do the same thing (it's not padded to two digits), but I think it's okay since this is just a debug message.